spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">The <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. For every literal of
+      any of these two datatypes, the value given as the <a>lexical form</a>
+      of the literal is rounded to a value that can be represented as an
```

Comment:
If we're going to be precise, it's not "rounded". That implies the lexical form has a value in and of itself. The mapping is defined to yield the IEEE 754-2008 value within the limits of IEEE 754-2008.

The point is simply that 

"0.1"^^xsd:float != "0.1"^^xsd:decimal != "0.1"^^xsd:double

Application beware.

```suggestion
      of the literal is a value that can be represented as an
```